### DISCUSSION TITLE
What solutions support Doccano Output? - for evaluation

### DISCUSSION THREAD

**Discussion by Diadriu** [doccano/doccano#1132] [Q&A]
First of all, I would like to thanks the developers of Doccano for making such a nice tool.
It has been of great use for my research project.

Now, I have two outputs of Doccano (it supports some kind of json or jsonl). The untouched input from my NER system, and the ground truth, both can be exported in 'json' or 'jsonl' output (which Doccano provide). Do you know which solutions support these outputs, in order to actually evaluate how well the NER system has performed? (I am sorry if this is a bit of topic, the compatibility of this format is what I am curious about)

Thank you.

**Comment by icoxfog417** [comment]
Thank you for using the doccano. We are very pleased to hear that doccano contributes to your research.

Does doccano-transformer become a suitable solution for you?
https://github.com/doccano/doccano-transformer
